PR: https://github.com/odoo/odoo/pull/97279

From: 3363e55cac2a121c29842a0b2463f940bdd866a4
From: Fabien Pinckaers
Date: 2022-08-01 22:26:53

Breaking data model changes score: 4, change matches:
-    reveal_ids = fields.Char(required=True,
+    reveal_ids = fields.Char(required=True)
-    available_state_ids = fields.One2many('res.country.state', compute='_compute_available_state_ids',
+    available_state_ids = fields.One2many('res.country.state', compute='_compute_available_state_ids')

Total Changes: 6

[IMP] cleanup of help messages in all modules

This commit aims at removing unuseful help message to:
1/ reduce translators work, to focus on more useful translations
2/ not sending unuseful information in load_views
3/ reduce help message to useful messages, so that we can mark
   fields having a tooltip in the future UI.
4/ some cleanup of existing messages too

The main use cases:
- REMOVED: help redundant with the field name, providing no extra info
- MOVED TO COMMENT: technical help messages, that should not be in UX

closes odoo/odoo#97279

Signed-off-by: Fabien Pinckaers <fp@odoo.com>

================================= pseudo patch: =================================

--- a/addons/crm_iap_mine/models/crm_iap_lead_industry.py
+++ b/addons/crm_iap_mine/models/crm_iap_lead_industry.py
@@ -11,8 +11,7 @@ class CrmIapLeadIndustry(models.Model):
     _order = 'sequence,id'
 
     name = fields.Char(string='Industry', required=True, translate=True)
-    reveal_ids = fields.Char(required=True,
-        help="The list of reveal_ids for this industry, separated with ','")
+    reveal_ids = fields.Char(required=True) # The list of reveal_ids for this industry, separated with ','
     color = fields.Integer(string='Color Index')
     sequence = fields.Integer('Sequence')
 

--- a/addons/crm_iap_mine/models/crm_iap_lead_mining_request.py
+++ b/addons/crm_iap_mine/models/crm_iap_lead_mining_request.py
@@ -61,8 +61,7 @@ class CRMLeadMiningRequest(models.Model):
     company_size_max = fields.Integer(default=1000)
     country_ids = fields.Many2many('res.country', string='Countries', default=_default_country_ids)
     state_ids = fields.Many2many('res.country.state', string='States')
-    available_state_ids = fields.One2many('res.country.state', compute='_compute_available_state_ids',
-        help="List of available states based on selected countries")
+    available_state_ids = fields.One2many('res.country.state', compute='_compute_available_state_ids')
     industry_ids = fields.Many2many('crm.iap.lead.industry', string='Industries')
 
     # Contact Generation Filter
